library/CoreBundle/Resources/config/parameters.yml

```diff
@@ -1,35 +1,32 @@
 # This file is auto-generated during the composer install
 imports:
-    - resource: '@CoreBundle/Resources/config/orm_soft_delete.yml'
+    -
+        resource: '@CoreBundle/Resources/config/orm_soft_delete.yml'
 parameters:
     database_host: data.ivozprovider.local
     database_port: null
     database_name: ivozprovider
     database_user: root
     database_password: changeme
-
     mailer_transport: smtp
     mailer_host: data.ivozprovider.local
     mailer_user: null
     mailer_password: null
     mailer_default_fromname: 'IvozProvider Notifications'
     mailer_default_fromaddress: no-replay@ivozprovider
     secret: ThisTokenIsNotSoSecretChangeIt
-
     ari_user: ivozariuser
     ari_password: ivozarisecret
     ari_port: 8088
-
     local_storage_path: /opt/irontec/ivozprovider/storage
     raw_recordings_dir: /opt/irontec/ivozprovider/storage/ivozprovider_model_recordings.originalfile/
-
     gearmand:
-      servers:
-        - 'jobs.ivozprovider.local'
-      client:
-        timeout: 5000
-      mail:
-        from: 'no-reply@irontec.com'
-        to:
-          - 'vozip@irontec.com'
-        subject: '[ERROR] XMLRPC'
+        servers:
+            - jobs.ivozprovider.local
+        client:
+            timeout: 5000
+        mail:
+            from: no-reply@irontec.com
+            to:
+                - vozip@irontec.com
+            subject: '[ERROR] XMLRPC'
```

Comment:
Is there a reason to use 4 spaces indentation here?

Also, I prefer to remove the _mail_ section, that should not be used (and in case it's used should not contain irontec addresses)

Reply:
Nevermind all the file uses 4 spaces 😅